packages/auth-compat/karma.conf.js

```diff
@@ -29,6 +30,15 @@ module.exports = function (config) {
     // frameworks to use
     // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
     frameworks: ['mocha'],
+    webpack: {
```

Comment:
Can we add some comments on these (here and in auth and functions)? Basically it's that undici is a fetch polyfill that test helpers call for Node tests, and browser tests should ignore that import to avoid compilation errors on the test helpers.

Reply:
Done!